lib/modules/dosomething/dosomething_global/dosomething_global.module

```diff
@@ -651,6 +651,24 @@ function dosomething_global_url($path, $options = NULL) {
 }
 
 /**
+ * Build a global node URL that adds the prefix and a fragment
+ *
+ * @param string $nid
+ *    The nid of the node
```

Comment:
I think we have a lot of cleaning up to do then, a lot of these comment clocks have no periods haha